app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @challenge = Challenge.find(@project.challenge_id)
```

Comment:
This works, but it's a bit overkill, considering a project `belongs_to ` a challenge - this means we can do `@project.challenge` and get the challenge instance that way.